src/styles/index.scss

```diff
@@ -1,3 +1,72 @@
+$cell-size: 100px;
+$default-month-length: 31;
+$days-of-week: (
+  'mon': '0',
+  'tue': '1',
+  'wed': '2',
+  'thu': '3',
+  'fri': '4',
+  'sat': '5',
+  'sun': '6',
+);
+$gap: 1px;
+
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+}
+
+.calendar {
+  gap: $gap;
+  display: flex;
+  flex-wrap: wrap;
+  width: 7 * ($cell-size + $gap) - $gap;
```

Comment:
The width calculation here seems to be a bit off. According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), calculations should be wrapped in parentheses to improve readability. Moreover, the calculation should account for the total number of gaps, which is one less than the number of cells (7 cells - 1 = 6 gaps). The correct calculation should be: `width: 7 * $cell-size + 6 * $gap;`.